mattermost-helm/charts/mattermost-app/templates/config.json

```diff
@@ -293,5 +293,12 @@
     "DataRetentionSettings": {
         "Enable": false
     }
+    {{- if .Values.jobserver.enabled -}}
+    ,
+    "JobSettings": {
+        "RunJobs": false,
+        "RunScheduler": false
+    }
+    {{- end -}}
 }
 {{- end -}}
```

Comment:
can you please add an empty line? here and in the other files as well :) thanks so much for this. I will run to validate that